go.mod

```diff
@@ -39,8 +39,8 @@ require (
 	github.com/oapi-codegen/runtime v1.1.1
 	github.com/operator-framework/api v0.27.0
 	github.com/operator-framework/operator-lifecycle-manager v0.27.0
-	github.com/percona/everest-operator v0.6.0-dev1.0.20250325152903-99872cc6005d
-	github.com/percona/percona-helm-charts/charts/everest v0.0.0-20250325153644-c152ee312dca
+	github.com/percona/everest-operator v0.6.0-dev1.0.20250326200335-220365110c38
```

Comment:
Update to the correct version once https://github.com/percona/everest-operator/pull/718 is merged to main.